internal/services/appservice/service_plan_resource_test.go

```diff
@@ -379,6 +394,36 @@ resource "azurerm_service_plan" "test" {
 `, data.RandomInteger, data.Locations.Primary, sku, count)
 }
 
+func (r ServicePlanResource) memoryOptimized(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-appserviceplan-%[1]d"
+  location = "%s"
+}
+
+resource "azurerm_service_plan" "test" {
+  name                     = "acctest-SP-%[1]d"
+  resource_group_name      = azurerm_resource_group.test.name
+  location                 = azurerm_resource_group.test.location
+  sku_name                 = "P1mv3"
+  os_type                  = "Linux"
+  per_site_scaling_enabled = true
+  worker_count             = 3
+
+  zone_balancing_enabled = true
+
+  tags = {
+    environment = "AccTest"
+    Foo         = "bar"
+  }
+}
+`, data.RandomInteger, data.Locations.Primary)
```

Comment:
Could you update this to:
```suggestion
`, data.RandomInteger, data.Locations.Secondary)
```
and I'll PR a change for the CI setup to support the testing of this SKU?

Reply:
Sure!